src/resolvers-cassandra/Settings/mutations.js

```diff
@@ -143,8 +143,11 @@ function createStream(args, res) { // eslint-disable-line no-unused-vars
   return new Promise((resolve, reject) => {
     const pipelineKey = args && args.input && args.input.pipelineKey;
     if (!pipelineKey || !pipelineKey.length) return reject('No pipelineKey specified.');
-    
-    const params = paramEntryToMap(args.input.params);
+
+    const paramEntry = args && args.input && args.input.params;
+    if (!paramEntry || !paramEntry.length || paramEntry.length == 0) return reject('No params specified.');
```

Comment:
If you make the param required, no need for this check as graphql will reject a request that doesn't contain the param.